test-utils/testlib/src/standard_test_cases.rs

```diff
@@ -840,6 +840,30 @@ pub fn test_delete_key_last(node: impl Node) {
     assert!(node_user.get_access_key(&account_id, &node.signer().public_key()).is_err());
 }
 
+fn assert_access_key(
+    access_key: &AccessKey,
+    access_key_view: AccessKeyView,
+    result: &FinalExecutionOutcomeView,
+    user: Box<dyn User>,
```

Comment:
```suggestion
    user: &dyn User,
```

seems like just ref is enough?